src/github.com/matrix-org/dendrite/clientapi/routing/register_test.go

```diff
@@ -145,3 +150,96 @@ func TestEmptyCompletedFlows(t *testing.T) {
 		t.Error("Empty Completed Flow Stages should be a empty slice: returned ", ret, ". Should be []")
 	}
 }
+
+// This method tests validation of the provided Application Service token and
+// username that they're registering
+func TestValidationOfApplicationServices(t *testing.T) {
+	// Set up application service namespaces
+	regex := "@_appservice_.*"
+	regexp, err := regexp.Compile(regex)
+	if err != nil {
+		t.Errorf("Error compiling regex: %s", regex)
+	}
+	fakeNamespace := config.ApplicationServiceNamespace{
+		Exclusive:    true,
+		Regex:        regex,
+		RegexpObject: regexp,
+	}
+	fakeNamespaceSlice := make([]config.ApplicationServiceNamespace, 1)
+	fakeNamespaceSlice[0] = fakeNamespace
```

Comment:
Can this not be done as a `fakeNameSlice := []config.ApplicationServiceNamespace{ fakeNamespace }`?